groups/instance/locals.tf

```diff
@@ -63,9 +51,25 @@ locals {
   ldap_group_settings_strategy               = local.secrets.ldap_group_settings_strategy
   ldap_group_settings_subtree                = local.secrets.ldap_group_settings_subtree
 
-  artifactory_license = local.secrets.artifactory_license
-
-  admin_password = local.secrets.admin_password
+  parameter_store_secrets = merge(
+    {
+      "db_username"                   = local.secrets.db_username
+      "db_password"                   = local.secrets.db_password
+      "ldap_setting_managerdn"        = local.secrets.ldap_setting_managerdn
+      "ldap_setting_manager_password" = local.secrets.ldap_setting_manager_password
+      "artifactory_license"           = local.secrets.artifactory_license
+      "admin_password"                = local.secrets.admin_password
+      "db_masterkey"                  = local.secrets.db_masterkey
+      "artifactory_access_token"      = local.secrets.artifactory_access_token
+    }
+  )
 
-  efs_access_point_id = module.efs_file_system.efs_access_point_ids["${var.efs_artifacts_access_point_name}"].id
+  db_username_param_name                   = "/${var.service}/${var.environment}/db_username"
```

Comment:
You could introduce a "base path" local - e.g. `param_base_path = "/${var.service}/${var.environment}"` and then reference that rather than repeating - `db_username_param_name = "${local.param_base_path}/db_username"`
Could also use that same local in parameter-store.tf & iam.tf too then.

Reply:
New local: param_base_path = "/${var.service}/${var.environment}"

Applied to locals.tf & parameter-store.tf

New local: base_path = "${var.service}-${var.environment}"

Applied to: 

	•	asg.tf
	•	iam.tf
	•	instance.tf
	•	kms.tf
	•	launch_template.tf
	•	locals.tf
	•	parameter-store.tf
	•	security_groups.tf

parameter-store.tf renamed to follow naming convention to parameter_store.tf